src/Illuminate/Notifications/NotificationServiceProvider.php

```diff
@@ -40,5 +40,7 @@ public function register()
         $this->app->alias(
             ChannelManager::class, FactoryContract::class
         );
+
+        $this->app->bind(SendQueuedNotifications::class, SendQueuedNotifications::class);
```

Comment:
You don't need to call `bind()` to bind a concrete class to itself. 

The container can handle it.

Reply:
Ah, right, good point. Removed.